docs/CONTRIBUTING.md

```diff
@@ -8,6 +8,10 @@ The following is a set of guidelines for contributing to SecDevLabs. These are j
 
 Well, a good place to start is to make sure you can deploy your app with Docker-Compose. All the apps in SecDevLabs are built with it in order to ensure you can get get right into the action! ✓
 
+If you need help with it, make sure to check [Docker Compose Getting Started][6]. There you'll find examples for both `Dockerfile` and `docker-compose.yml`, two essential files for deploying your app.
+
+For an even more detailed look at `docker-compose.yml` structure, you can check [Compose File Reference][7]. All of the existing apps have these files inside the deployments folder, so remember you can use them as example too!
```

Comment:
I'm afraid that adding this might be a little overwhelming for someone who has just started using Docker, what do you think @Takehime?

Reply:
I added Compose File Reference because the Getting Started guide doesn't explain much about the components of `docker-compose.yml` file, it only shows some examples, while the File Reference has a good explanation for all topics of the structure.